components/footer/Footer.tsx

```diff
@@ -34,53 +36,139 @@ function SectionItem({ item }: { item: Item }) {
           </div>
         )
         : (
-          <a href={item.href}>
+          <a
+            href={item.href}
+            class="max-md:text-sm text-xs font-normal text-neutral hover:text-accent transition-all duration-500"
+          >
             {item.label}
           </a>
         )}
     </span>
   );
 }
 
+export interface SecuritiesItem {
+  image: LiveImage;
+}
+
+export interface FooterImage {
+  image: LiveImage;
+  /**
+   * @title Label
+   */
+  label: string;
+}
+
 function FooterContainer(
   { children, class: _class = "" }: {
     class?: string;
     children: ComponentChildren;
   },
 ) {
-  return <div class={`py-6 px-4 sm:py-12 sm:px-0 ${_class}`}>{children}</div>;
+  return <div class={_class}>{children}</div>;
 }
 
 export interface Props {
   sections?: Section[];
+  /**
+   * @title Social Network
+   */
+  socialNetWorks: SocialItem[];
+  /**
+   * @title Phone number
+   */
+  phone: string;
+  /**
+   * @title E-mail
+   */
+  email: string;
+  /**
+   * @title Opening hours
+   * @format html
+   * @default Seg. à Sex. das 09:00h às 18:00h <br /> Sábado das 10:00h às 14:00h
+   */
+  openingHours: string;
+  /**
+   * @title Payments
+   */
+  payments?: PaymentItem[];
+  /**
+   * @title Poweredby
+   */
+  securities?: SecuritiesItem[];
+  /**
+   * @title Poweredby
+   */
+  poweredby?: FooterImage[];
+
+  /**
+   * @title Copy Right
+   * @default Agência N1 - Todos os Direitos Reservados
+   */
+  copyRight: string;
 }
 
-function Footer({ sections = [] }: Props) {
+function Footer(
+  {
+    sections = [],
+    socialNetWorks,
+    openingHours,
+    email,
+    phone,
+    payments,
+    securities,
+    poweredby,
+    copyRight,
+  }: Props,
+) {
   return (
-    <footer class="w-full bg-primary flex flex-col divide-y divide-primary-content">
-      <div>
-        <div class="container w-full flex flex-col divide-y divide-primary-content">
-          <FooterContainer>
-            <Newsletter />
-          </FooterContainer>
-
-          <FooterContainer>
-            {/* Desktop view */}
-            <ul class="hidden sm:flex flex-row gap-20">
+    <footer class="">
+      <div class="bg-white flex flex-col sm:max-w-[786px] lg:max-w-[1170px] xl:max-w-[1426px] 2xl:max-w-[1682px] w-full m-auto px-5">
+        <div class="flex items-start justify-start gap-8 mt-12 max-lg:flex-col">
+          <div
+            class="pt-11 px-8 pb-10 max-lg:w-full"
+            style={{
+              background: "#F7F7F7",
+              borderRadius: "10px",
+            }}
```

Comment:
O atruibuto style so pode ser usado para propriedades com variaçoes, de uma olhada no Container para um exemplo de caso.
```suggestion
            class="pt-11 px-8 pb-10 max-lg:w-full bg-[#f7f7f7] rounded-[10px]"
```